features/step_definitions/common.js

```
@@ -0,0 +1,20 @@
+const { Given, Then } = require('@cucumber/cucumber');
+const path = require('path'); // Add this line to import the path module
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)